popup.js

```diff
@@ -52,7 +58,9 @@ previouses.forEach((previous) => {
 });
 
 async function setShortcuts() {
+  
```

Comment:
There could be an error handling mechanism to handle if the 'getAll()' command falis.